features/in_onderzoek.feature

```diff
@@ -127,33 +127,32 @@ Functionaliteit: in onderzoek
       Gegeven het systeem heeft een persoon met de volgende gegevens
       | naam                | waarde    |
       | burgerservicenummer | 555550001 |
-      En de nationaliteit heeft volgende 'inOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | 047310   |
-      | datumIngangOnderzoek          | 20220307 |
+      En de persoon heeft een 'nationaliteit' met de volgende gegevens
+      | naam                                     | waarde   |
+      | aanduiding gegevens in onderzoek (83.10) | 047310   |
+      | datum ingang onderzoek (83.20)           | 20220307 |
       Als personen wordt gezocht met de volgende parameters
-      | naam                | waarde                              |
-      | type                | RaadpleegMetBurgerservicenummer     |
-      | burgerservicenummer | 555550001                           |
-      | fields              | burgerservicenummer,nationaliteiten |
-      Dan heeft de persoon met burgerservicenummer '555550001' GEEN 'inOnderzoek' gegevens
-      En heeft de nationaliteit GEEN 'inOnderzoek' gegevens
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550001                       |
+      | fields              | nationaliteiten                 |
+      Dan heeft de response een persoon met een 'nationaliteit' met de volgende gegevens
+      | naam | waarde                |
+      | type | NationaliteitOnbekend |
 
     @proxy
     Abstract Scenario: persoon heeft <gegeven in onderzoek> in onderzoek dat niet is gevraagd met fields
       Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 555550001 |
-      En de persoon heeft volgende 'inOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | <waarde> |
-      | datumIngangOnderzoek          | 20220307 |
+      | naam                                        | waarde    |
+      | burgerservicenummer                         | 555550001 |
+      | aanduiding gegevens in onderzoek (01.83.10) | <waarde>  |
+      | datum ingang onderzoek (01.83.20)           | 20220307  |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 555550001                       |
       | fields              | <fields>                        |
-      Dan heeft de persoon GEEN 'inOnderzoek' gegevens
+      Dan heeft de response een leeg persoon object
```

Comment:
Waarom heeft de response een leeg persoon object? Toch alleen als de uitgevraagde fields leeg zijn. Dat dit het geval is kan ik niet concluderen uit regel 146 t/m 149. Is het trouwens voor dit scenario van belang dat de uitgevraagde velden leeg zijn? Lijkt me niet en ik denk dan ook dat dit het scenario alleen maar onduidelijker maakt.

Reply:
De response heeft een persoon object omdat er een bestaand persoon wordt bevraagd. Dat de persoon object leeg is, is omdat de persoon de bevraagde velden niet heeft. Met 'Gegeven' stappen geeft je aan welke gegevens een persoon heeft. Wat niet wordt opgegeven, heeft de persoon dus niet of is niet relevant voor de betreffende scenario. Dit is inderdaad impliciet. Wil je het expliciet aangeven, dan kan dat zoals dat op sommige plekken (bijv. partners.feature) is gedaan. Maar als je dit wilt, dan moet je het naar mijn mening voor elke veld moeten doen en niet voor sommige velden wel en sommige niet